### PR TITLE
bump deps to latest versions

### PR DESCRIPTION
**Pull request by yjhmelody**
I find that `failure` version is over old.

### CODE REVIEW

File: cranelift-native/Cargo.toml [hunk 1]
```diff
@@ -14,7 +14,7 @@ cranelift-codegen = { path = "../cranelift-codegen", version = "0.45.0", default
 target-lexicon = "0.8.1"
 
 [target.'cfg(any(target_arch = "x86", target_arch = "x86_64"))'.dependencies]
-raw-cpuid = "6.0.0"
+raw-cpuid = "7.0"
```
Comment (bjorn3) [inline cranelift-native/Cargo.toml]: This is the only non semver compatible version bump.

Comment (yjhmelody) [reply cranelift-native/Cargo.toml]: The tests are passed. Need it downcast to 6.0 or 6.0.0 ?

**Comment by bnjbvr** [comment]
Thanks for opening a PR. We haven't really decided of what the proper dependency management for Cranelift should look like, so until we've discussed this (at least internally), I think we should put this PR (and other dependabot's PRs) on hold.

**Comment by bnjbvr** [comment]
@yjhmelody Did you have a specific motivation to bump the crates, that is, a new feature was introduced in some crate and you needed it for something in Cranelift, or there was an important bug fix / performance improvement that justified bumping a minor version of a specific crate? The "failure" crate is mentioned in the PR's description, but I would like to understand if there was a specific motivation to bump it, or others in general. If not, I'd prefer that we'd keep them as is to avoid dependency traffic (when vendoring dependencies) and imposing specific versions. Thanks!

**Comment by yjhmelody** [comment]
Sorry, no special motivation to bump version. Please feel free to deal with this pr.
@bnjbvr

**Comment by bnjbvr** [comment]
Ok, thanks for letting me know. I'll close this PR right now because I don't see a clear motivation to bump versions, but I appreciate you opening it in the first place. If you're looking for other contributions to get acquainted with the code base, feel free to have a look at the so-called [easy](https://github.com/CraneStation/cranelift/issues?q=is%3Aissue+is%3Aopen+label%3AE-easy) issues, or other issues, and let us know if there's something you'd like to give a try there!

**Pull request closed without merging**